data/ka'het missions.txt

```diff
@@ -0,0 +1,137 @@
+			`	"Then," she continues, "a few hundred years ago, a scout fleet arrived in a system where some Ka'het were still alive, which attacked the Remnant almost instantly and before they could activate the cloak some EMPs had already disabled two Starlings. They couldn't recover soon enough to survive, and the only remaining ship could do nothing but flee here as soon as possible."`
+			choice
+				`	"Which technology did you recover from the crashed Ka'het?"`
+					goto tech
+				`	"Wait, why did you call them 'shells' and not ships?"`
+					goto shells
+				`	"Have the Ka'het ever attacked the Remnant planets?"`
+			`	"Actually, we never met any Ka'het in the Ember Waste at all. The 'shells' we found had similar capabilities to the quantum keystone, meaning they could pass through the wormholes, so we don't know for sure why they remain there. Perhaps they aren't aware of their skills, or maybe they lost them over time."`
+			choice
+				`	"Wait, why did you call them 'shells' and not ships?"`
+					goto shells
+			label tech
+			`	"Back then we found large numbers of their EMP torpedoes, most of which in almost pristine conditions; their internal mechanisms were also simple enough we were able to reproduce them perfectly. We didn't find enough of the other systems in the 'shells' to do the same, but their tech still have been our main inspiration point for centuries."`
+			choice
+				`	"Wait, why did you call them 'shells' and not ships?"`
+			label shells
+			`	"You see, <first>," she says, "the Ka'het aren't normal spaceships. The debris we found were artificial exoskeletons, built around huge living beings; it's unlikely a species that big could have developed that sort of technology, so we presume someone must have created it for them. This could have happened in the Graveyard, since it's the only place where we know them to still live in, but we have such a limited data about that section of space that we can't say for sure." As she finishes talking, one of the people Taely's was talking with earlier comes closer and shows her something on a touchpad. "I have to go now, Captain," she tells you. "Good luck."`
+				decline
+	on decline
+		log "Asked Taely about the Graveyard and the Ka'het. She spoke of the previous contacts between them and the Remnant."
+		log "Factions" "Ka'het" `The Ka'het are aliens which live in an area south of the Ember Waste, called the Graveyard. They are large lifeforms which can travel in space using artificial exoskeletons and will attack anyone on sight, even sometimes other Ka'het.`
+
+mission "First Contact: Ka'het: Quarg 1"
+	name "Abandoned Ringworld"
+	description "Visit a Quarg station to ask them about the abandoned ring in the middle of Ka'het space."
+	landing
+	source "Querii Ka"
+	destination "Lagrange"
+		has "First Contact: Quarg: offered"
```

Comment:
Missing the `to offer` for this. I question if we want to prevent this mission from offering if the player hasn't already met the Hai, though. The Quarg first contact mission makes it clear that the player already knows what the Quarg are and what their ringworlds are, so I imagine the player would recognize the ringworld in the Graveyard and know that something is up since it's deserted. You'd just need to add a `branch` in the conversation based off of if the player `has "First Contact: Quarg: offered"` to change it.